url_launcher/plugin.go

```diff
@@ -0,0 +1,53 @@
+package url_launcher
+
+import (
+	"fmt"
+
+	"github.com/go-flutter-desktop/go-flutter"
+	"github.com/go-flutter-desktop/go-flutter/plugin"
+	"github.com/pkg/browser"
+	"github.com/pkg/errors"
+)
+
+const channelName = "plugins.flutter.io/url_launcher"
+
+type UrlLauncherPlugin struct{}
+
+var _ flutter.Plugin = &UrlLauncherPlugin{} // compile-time type check
+
+func (p *UrlLauncherPlugin) InitPlugin(messenger plugin.BinaryMessenger) error {
+	fmt.Println("InitPlugin")
+	channel := plugin.NewMethodChannel(messenger, channelName, plugin.StandardMethodCodec{})
+	channel.HandleFunc("launch", p.launch)
+	channel.HandleFunc("canLaunch", p.canLaunch)
+	return nil
+}
+
+func (p *UrlLauncherPlugin) launch(arguments interface{}) (reply interface{}, err error) {
+	fmt.Println("launch")
+	var url string
+
+	argsMap := arguments.(map[interface{}]interface{})
+	url = argsMap["url"].(string)
+	if url == "" {
+		return nil, errors.New("url is empty")
+	}
```

Comment:
Add WARNING message.
if  [`"useWebView"`](https://github.com/flutter/plugins/blob/0b8b6f4df57d56d8b86e3c5adcc5a7606d87f06e/packages/url_launcher/lib/url_launcher.dart#L59-L60) is set to true, print something like `plugins.flutter.io/url_launcher: WebView aren't supported on desktop`.

Reply:
keep the same logic, don't return an error, just a print